server/game/cards/characters/02/gendry.js

```diff
@@ -0,0 +1,58 @@
+const DrawCard = require('../../../drawcard.js');
+
+class Gendry extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onDominanceDetermined: (event, winner) => this.controller === winner
+            },
+            handler: () => {
+                this.modifyPower(1),
+                this.game.addMessage('{0} uses {1} gain a power on {2}', this.controller, this, this);
+            }
+        });
+
+        this.forcedReaction({
+            when: {
+                onDominanceDetermined: (event, winner) => this.game.getOtherPlayer(this.controller) === winner
```

Comment:
Change this to event, winner, loser, then check this.controller === loser

I will do the code to support this parameter as a separate task